manage-data/lifecycle/index-lifecycle-management/policy-apply.md

```diff
@@ -0,0 +1,72 @@
+---
+navigation_title: Manually apply a policy to an index
+applies_to:
+  stack: ga
+products:
+  - id: elasticsearch
+---
+
+# Manually apply a lifecycle policy to an index [apply-policy-manually]
+
+When you create new {{es}} index you can use an index template to apply the lifecycle policy by which the index will be managed. This process is described in [Configure a lifecycle policy](/manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md).
+
+You can also manually apply a lifecycle policy to an existing index, as described on this page. If an index is currently managed by an ILM policy you must first remove that policy before applying a new one. Refer to [Switch to a different lifecycle policy](/manage-data/lifecycle/index-lifecycle-management/policy-updates.md#switch-lifecycle-policies) for details.
+
+You can do this procedure in {{kib}} or using the {{es}} API.
+
+::::{warning}
+Do not manually apply a policy that uses the rollover action to an index which has not yet rolled over. Otherwise, the policy may not be carried forward when the rollover action creates a new index.
```

Comment:
```suggestion
Do not manually apply a policy that uses the rollover action. Policies that use rollover must be applied by the [index template](#apply-policy-template). Otherwise, the policy is not carried forward when the rollover action creates a new index.
```

Reverting to original text.